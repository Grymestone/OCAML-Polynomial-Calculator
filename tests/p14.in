76a^1 + 56a^20 + (2a^2 + 3a)
