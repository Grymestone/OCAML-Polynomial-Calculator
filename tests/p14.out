(((76*(a)^1)+(56*(a)^20))+((2*(a)^2)+(3*a)))
2x^2 + 56x^20 + 79x^1